### PR TITLE
show mobile app download from web dashboard under flag

### PR DESCRIPTION
**Pull request by czue**
https://trello.com/c/nr91ickz/291-host-v1-release-ready-apk-on-icdsgov

image

This is mostly refactors / cleanup. Functional change is in https://github.com/dimagi/commcare-hq/commit/643ce76e1e40d1aec6b61ca607107f88a9c31e56.

I'd really rather not keep the APK file in the repo, but the team wants this to go out on today's deploy, so doing it this way as a stopgap. This is also how the user manual works (which I'd also prefer to change).

Review how you like.

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -989,6 +989,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
     [NAMESPACE_USER]
 )
 
+ICDS_DASHBOARD_SHOW_MOBILE_APK = DynamicallyPredictablyRandomToggle(
+    'icds_dashboard_show_mobile_apk',
+    'Show a "Mobile APK" download link on the ICDS Dashboard',
```
Comment (Rohit25negi) [inline corehq/toggles.py]: just for my knowledge and might be completely unrelated. This mobile APK is the android app which renders the mobile dashboard views right?

File: custom/icds_reports/templates/icds_reports/dashboard.html [hunk 2]
```diff
@@ -132,50 +133,59 @@ <h2 class="dashboard-title">
...
                                                       <i class="fa fa-question-circle nav-main-icon fa-2x light-grey"
                                                           uib-popover-html="'Download user manual'"
                                                           popover-placement="left"
-                                                          popover-trigger="'mouseenter'" style="margin-top: 11px;"></i>
+                                                          popover-trigger="'mouseenter'" ></i>
                                                   </a>
                                               </span>
                                           </li>
-                                          <li class="dropdown" ng-hide="$location.path().indexOf('download') !== -1">
+                                      {% if show_mobile_apk %}
```
Comment (Rohit25negi) [inline custom/icds_reports/templates/icds_reports/dashboard.html]: for my own knowledge. Is there a reason it has to use its separate feature flag and cant use this general one https://www.icds-cas.gov.in/hq/flags/edit/features_in_dashboard_icds_reports/

**Review by Rohit25negi — APPROVED** [review]
looks good. Left two questions. But not blocking. One for purely my knowledge and the second one for maybe code clarity(but maybe invalid)

File: custom/icds_reports/static/css/variables.less [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+
+// Z-Indices
+
+@zindexDropdownMenu: 4;
```
Comment (orangejenny) [inline custom/icds_reports/static/css/variables.less]: 🥇

File: corehq/toggles.py (hunk 1 above)
Comment (czue) [reply corehq/toggles.py]: Yep, exactly

File: custom/icds_reports/templates/icds_reports/dashboard.html (hunk 2 above)
Comment (czue) [reply custom/icds_reports/templates/icds_reports/dashboard.html]: I suspect that we'll want to "release" it to everyone at some unknown point in the future, and if it's a `DynamicallyPredictablyRandomToggle` then we can do that without a deploy.

**Pull request merged**